package/suse-migration-sle15-activation-spec-template

```diff
@@ -29,14 +29,9 @@ BuildArch:        noarch
 Conflicts:        suse-migration-services
 BuildRequires:    grub2
 BuildRequires:    python3-devel
-BuildRequires:    python3-setuptools
-BuildRequires:    python3-Cerberus
-BuildRequires:    python3-PyYAML
 Requires:         SLES15-Migration
+Requires:         suse-migration-pre-checks
```

Comment:
Requires(post)

Reply:
the requires of suse-migration-pre-checks can be removed here  since it is inherited from SLES15-Migration